build.gradle

```diff
@@ -0,0 +1,190 @@
+plugins {
+	id 'java'
+	id 'war'
+	id 'jacoco'
+	id 'org.ajoberstar.grgit' version '4.1.1'
```

Comment:
See https://andrewoberstar.com/posts/2024-04-02-dont-commit-to-grgit/

I would find an alternative solution.

Reply:
From reading that it sounds like he author isn't updating it any more, but there's no security issues or anything. Currently it Just Works so I think it's fine for now, and we can switch to JGit or a command line call if / when things break

Reply:
Also it's the way every other Gradle repo works so when we switch we should switch them all

Reply:
Sounds reasonable.